src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -26,7 +27,8 @@ $replicasetdetails-table-cell: #777;
 
 .kd-replicasetdetail-sidebar {
   background-color: $replicasetdetails-sidebar-bg;
-  min-width: 315px;
+  max-width: $replicasetdetails-sidebar-width;
```

Comment:
Ops actually I've set `$replicasetdetails-sidebar-width` variable to `315` instead of `315px`.